### PR TITLE
 Some settings now disable when the tournament is started

### LINKED ISSUE #15: Some settings should not be editable after the tournament has started
These settings are:
- The tiebreaker rule(s)
- The amount of stages
- The order of stages
- The seeding method (not added yet)
- The amount of teams (bots on the team can change, that is no problem right now)

### PR DESCRIPTION
**Pull request by NicEastvillage**
Settings that does not make sense to change after tournament start is now disabled in UI after start of tournament. Previously they did nothing and in some cases threw exceptions.

Resolves #15

### CODE REVIEW

File: src/dk/aau/cs/ds306e18/tournament/ui/ParticipantSettingsTabController.java [hunk 1]
```diff
@@ -372,16 +364,20 @@ private void clearBotFields() {
...
         // Handle team order button disabling / enabling
-        swapUpTeam.setDisable(seedingOption != SeedingOption.SEED_BY_ORDER || selectedIndex <= 0);
-        swapDownTeam.setDisable(seedingOption != SeedingOption.SEED_BY_ORDER || selectedIndex == teamsListView.getItems().size() - 1);
+        swapUpTeam.setDisable(seedingOption != SeedingOption.SEED_BY_ORDER || selectedIndex <= 0 || started);
+        swapDownTeam.setDisable(seedingOption != SeedingOption.SEED_BY_ORDER || selectedIndex == teamsListView.getItems().size() - 1 || started);
 
-        if (selectedIndex == -1) {
-            removeTeamBtn.setDisable(true);
-        }
+        System.out.println("TEST: " + (selectedIndex == -1 || started));
```
Comment (cogitantium) [inline src/dk/aau/cs/ds306e18/tournament/ui/ParticipantSettingsTabController.java]: Whoops, seems like a print-test got shipped.

File: src/dk/aau/cs/ds306e18/tournament/ui/RLBotSettingsTabController.java [hunk 2]
```diff
@@ -16,19 +16,20 @@
...
+    @FXML private Button configPathBtn;
+    @FXML private TextField configPathTextField;
+    @FXML private CheckBox autoCloseRLBotCheckBox;
+
 
     final private FileChooser fileChooser = new FileChooser();
 
     @FXML
     private void initialize() {
+        instance = this;
```
Comment (cogitantium) [inline src/dk/aau/cs/ds306e18/tournament/ui/RLBotSettingsTabController.java]: Is this not redundant, I don't see references or changes related to this elsewhere.

File: src/dk/aau/cs/ds306e18/tournament/ui/TournamentSettingsTabController.java [hunk 3]
```diff
@@ -40,15 +42,10 @@
 
     @FXML
     private void initialize() {
+        instance = this;
```
Comment (cogitantium) [inline src/dk/aau/cs/ds306e18/tournament/ui/TournamentSettingsTabController.java]: Same question as to second comment.

File: src/dk/aau/cs/ds306e18/tournament/ui/RLBotSettingsTabController.java (hunk 2 above)
Comment (NicEastvillage) [reply src/dk/aau/cs/ds306e18/tournament/ui/RLBotSettingsTabController.java]: It is referenced in the MainController.

**Pull request merged**